feat(content): Twin Fixed Blasters (Regular and Modified) (#7402)
endless-sky/endless-sky: data/human/weapons.txt

		"hull damage" 6.6
	description "Although not the most accurate or damaging of weapons, the Energy Blaster is popular because it is small enough to be installed on even the tiniest of ships. One blaster is not enough to do appreciable damage to anything larger than a fighter, but a ship equipped with several of them becomes a force to be reckoned with."

outfit "Twin Blaster"
	category "Guns"
	cost 40000
	thumbnail "outfit/twin blaster"
	"mass" 12
	"outfit space" -12
	"weapon capacity" -12
	"gun ports" -1
	weapon
		sound "blaster"
		"submunition" "Blaster Submunition"

> 			"offset" 3 0
> 		"submunition" "Blaster Submunition"
> 			"offset" -3 0

Reviewer: Six pixels apart looks very odd to me. At that distance it doesn't even look like the projectiles are coming out of the gun.
![image](https://user-images.githubusercontent.com/17688683/196828140-083c3f4c-d294-45a6-8bad-0c32bff20775.png)

I'd prefer something closer, like 3 pixels apart. At that distance it's about the thickness of the heavy laser turret, and looks more feasible from the point of the barrel.
![image](https://user-images.githubusercontent.com/17688683/196828164-0f1d55a1-7fb9-45b0-aa54-26e04ed8ae5e.png)
```suggestion
			"offset" 1.5 0
		"submunition" "Blaster Submunition"
			"offset" -1.5 0
```
Alternatively we can just double the fire rate as is done on the turrets.

> 			"offset" 1.5 0
> 		"submunition" "Blaster Submunition"
> 			"offset" -1.5 0

Reviewer: New proposal from @Hecter94: have the offsets be close at the barrel, but give each projectile a facing able so that they spread out over range.
```suggestion
			"offset" 1 0
			"facing" 0.3
		"submunition" "Blaster Submunition"
			"offset" -1 0
			"facing" -0.3
```

		"hull damage" 6.6
	description "A Quad Blaster Turret carries four guns on a single turret mount, doubling the firepower of an ordinary Blaster Turret for ships with enough space to install one and energy to drive it."

outfit "Twin Blaster"
	category "Guns"
	cost 40000
	thumbnail "outfit/twin blaster"
	"mass" 10
	"outfit space" -10
	"weapon capacity" -10

> 	"gun ports" -1

Reviewer 2: ```suggestion
	"mass" 13
	"outfit space" -13
	"weapon capacity" -13
	"gun ports" -1
```

This put its DPS/mass at 13.23, which is between the Proton Gun at 14.86 and the Particle Cannon at 12.7, which seems about right. 
Currently, its DPS/mass is above the heavy laser and just below the beam laser, which doesn't seem appropriate.

This also puts it directly between the Beam Laser at 8t and the Heavy Laser at 15t

	weapon
		sprite "projectile/2x blaster"
		sound "blaster"
		"hit effect" "blaster impact"
		"inaccuracy" 3
		"velocity" 10.625
		"lifetime" 48
		"reload" 12
		"firing energy" 11.6
		"firing heat" 36
		"shield damage" 21.2
		"hull damage" 13.2

> 	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of twice the heat and power consumption."

Reviewer 3: ```suggestion
	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of twice the heat and power consumption, plus a modicum of added weight.
```
Doesn't have to be this exactly, but if we're specifically calling out the exchange, I'd like to head off a million and one "bug reports" about the appropriate weight increase.

> 	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster, at the cost of twice the heat and power consumption."

Reviewer 4: ```suggestion
	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of twice the heat and power consumption."
```

> 	description "The combination of the Energy Blaster's ubiquity, low price, and stable performance has made it irresistable to tinkerers from the day it was first introduced. This particular pair of blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster, at the cost of producing twice the heat and consuming twice the power."

Reviewer 5: ```suggestion
	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster, at the cost of twice the heat and power consumption."
```

		"hull damage" 6.6
	description "A Quad Blaster Turret carries four guns on a single turret mount, doubling the firepower of an ordinary Blaster Turret for ships with enough space to install one and energy to drive it."

outfit "Twin Blaster"
	category "Guns"
	cost 40000
	thumbnail "outfit/twin blaster"
	"mass" 12
	"outfit space" -12
	"weapon capacity" -12
	"gun ports" -1
	weapon
		sound "blaster"
		"submunition" "Blaster Submunition"
			"offset" 3 0
		"submunition" "Blaster Submunition"
			"offset" -3 0
		"inaccuracy" 3
		"lifetime" 0
		"reload" 12
		"firing energy" 13.9
		"firing heat" 43.2

> 	description "The Energy Blaster's ubiquity, low price, and stable performance have made it irresistible to tinkerers from the day it was first introduced. These blasters have been bolted together to fit within a single gun port, offering twice the damage of a single blaster at the cost of a little more than twice the heat, mass, and power consumption."

Reviewer 5: Oops, I don't think we need two descriptions

		"hull damage" 8.8
	description "This is an Energy Blaster that has been tampered with to increase its power slightly, at the cost of reduced range and higher energy consumption. Modified Blasters also produce an extreme amount of heat when firing, increasing the risk that your ship will overheat."

outfit "Twin Modified Blaster"
	category "Guns"
	cost 84000
	thumbnail "outfit/twin mod blaster"
	"mass" 14
	"outfit space" -14
	"weapon capacity" -14
	"gun ports" -1
	weapon
		sound "mod blaster"
		"submunition" "Modified Blaster Submunition"

> 			"offset" 3 0
> 		"submunition" "Modified Blaster Submunition"
> 			"offset" -3 0

Reviewer: ```suggestion
			"offset" 1.5 0
		"submunition" "Modified Blaster Submunition"
			"offset" -1.5 0
```

> 			"offset" 1.5 0
> 		"submunition" "Modified Blaster Submunition"
> 			"offset" -1.5 0

Reviewer: ```suggestion
			"offset" 1 0
			"facing" 0.45
		"submunition" "Modified Blaster Submunition"
			"offset" -1 0
			"facing" -0.45
```

		"hull damage" 8.8
	description "This is a turreted version of the Modified Blaster, which is an Energy Blaster modified for greater power at the cost of higher energy consumption and heat. These turrets are popular with pirates and others who are trying to cram as much firepower into their ships as possible."

outfit "Twin Modified Blaster"
	category "Guns"
	cost 84000
	thumbnail "outfit/twin mod blaster"
	"mass" 12
	"outfit space" -12

> 	"weapon capacity" -12

Reviewer 2: ```suggestion
	"mass" 15
	"outfit space" -15
	"weapon capacity" -15
```

This puts it at the same weight as a heavy laser, with 26 less total DPS, but with more range, and less overall drain, which feels like an appropriate tradeoff. Especially considering the effective DPS of the heavy laser is likely to be even higher since it takes only a single frame to hit and is more accurate overall.

	"gun ports" -1
	weapon
		sprite "projectile/2x mod blaster"
		sound "mod blaster"
		"hit effect" "blaster impact"
		"inaccuracy" 4
		"velocity" 10
		"lifetime" 48
		"reload" 12
		"firing energy" 13.0
		"firing heat" 50.4
		"shield damage" 26.4
		"hull damage" 17.6

> 	description "An enterprising engineer, presumably having decided that the Modified Blaster's already-frightening lack of safety control was insufficent, has apparently taken two of the weapons and shoved them together in the same gun port, crudely welded together with fervid enthusiasm. Whether or not this weapon will destroy the target before it melts its own components is anyone's guess; perhaps the intent was that the twin barrels would intimidate enemies into submission so that the weapon would never have to be used."

Reviewer 5: ```suggestion
	description "An enterprising engineer, presumably having decided that the Modified Blaster's already-frightening lack of safety control was insufficient, has taken two of the weapons and shoved them together in the same gun port, crudely welded together with fervid enthusiasm. Whether or not this weapon will destroy the target before it melts its own components is anyone's guess; perhaps the intent was that the twin barrels would intimidate enemies into submission so that the weapon would never have to be used."
```
